tracer/missing-nullability-files.csv

```diff
@@ -170,10 +169,16 @@ src/Datadog.Trace/HttpOverStreams/HttpMessage.cs
 src/Datadog.Trace/HttpOverStreams/HttpRequest.cs
 src/Datadog.Trace/HttpOverStreams/HttpResponse.cs
 src/Datadog.Trace/HttpOverStreams/IHttpContent.cs
+<<<<<<< HEAD
```

Comment:
I think you accidentally included the git merge conflict

Reply:
Thanks for the heads up! Fixed.